Artesian/Artesian.SDK/Service/Query/IQueryWithExtractionInterval.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) ARK LTD. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for
+// license information. 
+using NodaTime;
+
+namespace Artesian.SDK.Service
+{
+    interface IQueryWithExtractionInterval<T>: IQuery<T>
+    {
+        T InRelativePeriod(Period extractionPeriod);
```

Comment:
Shouldnt this be in the other interface? Periods should be supported by Auction, while Intervals not.

Reply:
It seems Auction supports only Absolute time window and Relative Period Range and not Interval or Period

Reply:
Please rise this with Alessandro since it should be there, server-side.

In any case Period and Interval are two separate sets of capabilities, so you can safely split them this way while Alessandro expose that his side.

Reply:
Ok will do.